src/ServiceFabric/ServiceFabric.Test/ScenarioTests/Common.ps1

```diff
@@ -47,7 +47,7 @@ function Get-ResourceGroupName
 
 function Get-ClusterName
 {
-    return "azurermsfclustertest";
+    return "azurermsfclustertest2";
```

Comment:
Updating the shared cluster name failed the existing test cases as the name is part of some endpoints. You can use a new cluster name in the test case that needs re-record only.

Reply:
I updated this. Please take a look